lib/KTable.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+
+  <div class="k-table">
+    <table>
+      <thead>
+        <tr>
+          <th v-for="(header,index) in headers" :key="index">
+            {{ header }}
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(row,index) in rows" :key="index">
+          <td v-for="(cell,cellIndex) in row" :key="cellIndex">
+            {{ cell }}
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+
+</template>
+
+
+<script>
+
+  export default {
+    name: 'KTable',
+    props: {
+      headers: {
+        type: Array,
+        required: true,
+      },
+      rows: {
+        type: Array,
+        required: true,
+      },
+    },
+  };
+
+</script>
+
+
+<style scoped>
+.k-table{
+    margin:20px;
+    border-collapse:collapse;
+}
+.k-table table{
+    width:100%;
+    border:2px solid #ccc;
```

Comment:
No changes required, just a note that when we start getting to the final styles, hardcoded colors will never be used. Instead, tokens or palette colors will be used via computed styles as described at https://design-system.learningequality.org/colors. This will ensure that other themes will work (Kolibri can be themed)

Reply:
Okay, noted!